api/ConvolverNode.json

```diff
@@ -0,0 +1,209 @@
+{
+  "api": {
+    "ConvolverNode": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/ConvolverNode",
+        "support": {
+          "webview_android": {
+            "version_added": "14"
```

Comment:
We had a null here for the other Web Audio. If this one is correct, I'm all to fix the other WebAudio entries to have the "14" there too :-)

[Valid for the 2 members below too]

Reply:
Yes, I think this should be 14 for all the other instances. I will fix these soonish.

Reply:
\o/. I'm merging this one.